robottelo/ui/contenthost.py

```diff
@@ -112,6 +112,23 @@ def validate_subscription_status(self, name, expected_value=True,
                 return True
         return False
 
+    def get_subscription_color_and_status(self, name):
+        """Return content host subscription status color and name
+
+        :returns a tuple (color, status_name) or None
+        """
+        self.search(name)
+        sub_element = self.wait_until_element(
+            locators['contenthost.subscription_status'] % name)
+        if sub_element:
+            # class attribute string value looks like:
+            # "yellow host-status pficon pficon-info status-warn"
```

Comment:
so test case will fail, in case element class will change to:
`"something new host-status pficon pficon-info yellow status-warn"`
?

Reply:
@oshtaier yes we have no other way to retrieve the color

Reply:
Here is an example how you can do it:

![image](https://user-images.githubusercontent.com/10895065/33069417-29cc6b92-cebd-11e7-92b1-ff3f95fdbedb.png)

```
self.hosts.wait_until_element(('xpath', "//a[contains(., 'dft')]/preceding-sibling::span")).value_of_css_property('color')
```
rgba(63, 156, 53, 1)

```
self.hosts.wait_until_element(('xpath', "//a[contains(., 'linda')]/preceding-sibling::span")).value_of_css_property('color')
```
rgba(204, 0, 0, 1)

If 'color' property doesn't work for you, please also check 'background-color'.